packages/amplication-client/src/VersionControl/PendingChange.tsx

```diff
@@ -1,34 +1,55 @@
-import { Tooltip } from "@amplication/design-system";
-import classNames from "classnames";
-import React, { useMemo } from "react";
-import * as models from "../models";
-import "./PendingChange.scss";
-import PendingChangeContent from "./PendingChangeContent";
-import PendingChangeServiceTopics from "./PendingChangeServiceTopics";
+/* eslint-disable @typescript-eslint/no-unused-vars */
+import { Tooltip } from '@amplication/design-system';
+import classNames from 'classnames';
+import React, { useMemo, useState } from 'react';
+import * as models from '../models';
+import './PendingChange.scss';
+import PendingChangeContent from './PendingChangeContent';
+import PendingChangeServiceTopics from './PendingChangeServiceTopics';
+import { gql, useQuery } from '@apollo/client';
 
-const CLASS_NAME = "pending-change";
-const TOOLTIP_DIRECTION = "ne";
+const CLASS_NAME = 'pending-change';
+const TOOLTIP_DIRECTION = 'ne';
 
 type entityLinkAndDisplayName = {
   relativeUrl: string;
   icon: string;
   displayName: string;
 };
 
+type TData = {
+  Topics: models.Topic[];
+};
+
 type Props = {
+  topicIndex: number;
   change: models.PendingChange;
   linkToOrigin?: boolean;
 };
 
+const DATE_CREATED_FIELD = 'createdAt';
+
 const ACTION_TO_LABEL: {
   [key in models.EnumPendingChangeAction]: string;
 } = {
-  [models.EnumPendingChangeAction.Create]: "C",
-  [models.EnumPendingChangeAction.Delete]: "D",
-  [models.EnumPendingChangeAction.Update]: "U",
+  [models.EnumPendingChangeAction.Create]: 'C',
+  [models.EnumPendingChangeAction.Delete]: 'D',
+  [models.EnumPendingChangeAction.Update]: 'U',
 };
 
-const PendingChange = ({ change, linkToOrigin = false }: Props) => {
+const PendingChange = ({ topicIndex, change, linkToOrigin = false }: Props) => {
+  const { data: topicsData } = useQuery<TData>(GET_TOPICS, {
+    variables: {
+      where: {
+        resource: { id: 'cl9kzud3f24405wul5cj4u55f' },
+        displayName: undefined,
+      },
+      orderBy: {
+        [DATE_CREATED_FIELD]: models.SortOrder.Asc,
+      },
+    },
+  });
```

Comment:
The PendingChange component should not know anything specific about Topics.
Also, please note that you used hard-coded value of resourceId

I am rejecting this change as it is. The fix should be implemented in the Topic 
see this as an example 
https://github.com/amplication/amplication/blob/ed80357733cc315eaa7a8da92f6d7f6313f9bce2/packages/amplication-client/src/Plugins/hooks/usePlugins.ts#L179

Reply:
Do you mean I can change the resource id to ```data.setPluginOrder.id```

Reply:
I mean that you need to use `AddBlock` in `Topic.tsx` and `NewTopic.tsx`

Reply:
Okay, Thanks

Reply:
any updates? @SayyidMuhammad878

Reply:
@SayyidMuhammad878  all good. We understand you may be busy. Let us know if you need us to take it over.

Reply:
I've fixed it you can check it out